public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js

```diff
@@ -180,31 +182,55 @@ class DefineMonitor extends Component {
   }
 
   async getSettings() {
+    const { httpClient } = this.props;
+    let remoteMonitoringEnabled = false;
+    let canCallGetRemoteIndexes = false;
+
+    // Check whether remote monitoring is enabled
     try {
-      const { httpClient } = this.props;
       const response = await httpClient.get('../api/alerting/_settings');
       if (response.ok) {
         const { defaults, transient, persistent } = response.resp;
-        let remoteMonitoringEnabled = _.get(
+        remoteMonitoringEnabled = _.get(
           // If present, take the 'transient' setting.
           transient,
-          REMOTE_MONITORING_ENABLED_SETTING_PATH,
+          CROSS_CLUSTER_MONITORING_ENABLED_SETTING,
           // Else take the 'persistent' setting.
           _.get(
             persistent,
-            REMOTE_MONITORING_ENABLED_SETTING_PATH,
+            CROSS_CLUSTER_MONITORING_ENABLED_SETTING,
             // Else take the 'default' setting.
-            _.get(defaults, REMOTE_MONITORING_ENABLED_SETTING_PATH, false)
+            _.get(defaults, CROSS_CLUSTER_MONITORING_ENABLED_SETTING, false)
           )
         );
-        // Boolean settings are returned as strings (e.g., `"true"`, and `"false"`). Constructing boolean value from the string.
-        if (typeof remoteMonitoringEnabled === 'string')
+
+        // Boolean settings can be returned as strings (e.g., `"true"`, and `"false"`). Constructing boolean value from the string.
+        if (typeof remoteMonitoringEnabled === 'string') {
           remoteMonitoringEnabled = JSON.parse(remoteMonitoringEnabled);
-        this.setState({ remoteMonitoringEnabled: remoteMonitoringEnabled });
+        }
       }
     } catch (e) {
-      console.log('Error while retrieving settings', e);
+      console.log('Error while retrieving settings:', e);
     }
+
+    // Check whether the user can call GetRemoteIndexes
+    if (remoteMonitoringEnabled) {
+      try {
+        const query = {
+          indexes: '*,*:*',
+          include_mappings: false,
+        };
+        const response = await httpClient.get(`../api/alerting/remote/indexes`, { query: query });
+        if (response.ok) canCallGetRemoteIndexes = true;
+      } catch (e) {
+        console.warn('Error while retrieving clusters:', e);
+      }
+    }
+
+    this.setState({
+      remoteMonitoringEnabled: remoteMonitoringEnabled,
```

Comment:
no need to specify value field when it is same as key
```
{
  remoteMonitoringEnabled,
  ...
}
```

Reply:
Ah, I typically use the `remoteMonitoringEnabled: remoteMonitoringEnabled` style for clarity. I'll revise this to your suggestion.